### DISCUSSION TITLE
Is there a tool to convert .Rmd files to .qmd?

### DISCUSSION THREAD

**Discussion by dmbates** [quarto-dev/quarto-cli#10723] [Q&A] [answered]
### Description

I feel that I may have missed something obvious.  I am looking for a tool to convert an `.Rmd` file to a `.qmd` file, including reformatting the options on code chunks.  Is there such a tool?

**Comment by cderv** [comment, marked as answer]
There is no tool to convert Rmd to Qmd completely (like converting `output:` to equivalent `format:`. 

But **knitr** has a tool to convert cell options. Look at `?knitr::convert_chunk_header` help page.

**Comment by danmackinlay** [comment]
I [wrote a script for my own migration](https://danmackinlay.name/notebook/quarto_websites.html#migrating-from-blogdown) which handled the header metadata, the citations, and supporting bibliography files better

**Reply by cscheid** [reply]
This is great! (cc @cderv, since we have discussed doing something like this ourselves in external tools)

**Reply by cderv** [reply]
Really cool @danmackinlay ! Thanks for sharing ! 

I want to add such function in **quarto** R package for all king of Rmd document so I may take inspiration from your script if that is ok !

**Reply by danmackinlay** [reply]
More than OK, please use it for whatever you want.  I'll update the site to reflect that the script is MIT licensed.

**Reply by danmackinlay** [reply]
Done. Made it into [a gist](https://gist.github.com/danmackinlay/c6f277e244de90e10b535efbe0610a0c)
